060_nineDivisors.js

```diff
@@ -0,0 +1,17 @@
+numbersWithNineDivisors = (n) => {
+  let nineDivisor = [];
```

Comment:
variable name can be plural depending on the value it is holding. 
Mostly we use plural name for arrays

Reply:
noted, changed array name.